ocs/ocp.py

```diff
@@ -65,6 +65,7 @@ def exec_oc_cmd(self, command):
 
         oc_cmd += command
         out = run_cmd(cmd=oc_cmd)
+        log.debug(out)
```

Comment:
We can ignore this change. The run_cmd function already has a ### log.debug(f"CMD output: {r.stdout.decode()}". Hence this is redundant.